server/schedule/filter/filters.go

```diff
@@ -26,6 +26,8 @@ import (
 
 // Filter is an interface to filter source and target store.
 type Filter interface {
+	// ActOn is used to indicate where the filter will act on.
+	ActOn() string
```

Comment:
I think using verb phrases as variable names seems to be inappropriate. But I have no good idea about it.